### PR TITLE
cargo-{help,version}: add page

### PR DESCRIPTION
**Pull request by wangme88**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/cargo-help.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# cargo help
+
+> Display help information about the Rust project manager.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-help.html>.
```
Comment (acuteenvy) [inline pages/common/cargo-help.md:4]:
```suggestion
> Display help on `cargo` and its subcommands.
> More information: <https://doc.rust-lang.org/cargo/commands/cargo-help.html>.
```

File: pages/common/cargo-help.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Display general help:
+
+`cargo help`
+
+- Display help for the `cargo help` subcommand:
+
+`cargo help --help`
```
Comment (acuteenvy) [inline pages/common/cargo-help.md:12]:
```suggestion
```

File: pages/common/cargo-version.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# cargo version
+
+> Display Cargo version information.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-version.html>.
+
+- Display Cargo version:
```
Comment (acuteenvy) [inline pages/common/cargo-version.md:6]:
```suggestion
- Display the version of `cargo`:
```

File: pages/common/cargo-version.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# cargo version
+
+> Display Cargo version information.
```
Comment (acuteenvy) [inline pages/common/cargo-version.md:3]:
```suggestion
> Display `cargo` version information.
```

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and Thanks for your contribution. The changes LGTM after @acuteenvy's suggestions.

**Comment by wangme88** [comment]
Thank you all for the suggestions and feedback

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**